source/features/cross-deleted-pr-branches.tsx

```diff
@@ -0,0 +1,62 @@
+import select from 'select-dom';
+import features from '../libs/features';
+import * as pageDetect from '../libs/page-detect';
+
+function inPR(): void {
+	let deletedBranch: string | undefined;
+	const lastBranchAction = select.all(`
```

Comment:
```suggestion
	const lastBranchAction = select.last(`
```

And you can drop `.pop`

Reply:
Added to new commit.